contracts/upgradeable_contracts/Ownable.sol

```diff
@@ -23,6 +24,20 @@ contract Ownable is EternalStorage {
         _;
     }
 
+    /**
+    * @dev Throws if called by any account other than contract itself or owner.
+    */
+    modifier onlyRelevantSender() {
+        // proxy owner if used through proxy, address(0) otherwise
+        require(
+            !address(this).call(abi.encodeWithSignature("upgradeabilityOwner()")) || // covers usage without calling through storage proxy
```

Comment:
I have done some tests. Seems like compiler optimises such things, so that contract does not store string literal, and stores pre-calculated selector instead. Contract bytecodes are identical in both cases.
But for consistency with other code parts, resolved this in bc189a3.

Reply:
thanks. Just FYI the previous activity for replacing string literals by hashes was done under https://github.com/poanetwork/tokenbridge-contracts/pull/303.